apis/fluentbit/v1alpha2/plugins/output/prometheus_remote_write_types.go

```diff
@@ -0,0 +1,99 @@
+package output
+
+import (
+	"fmt"
+
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins"
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins/params"
+	"github.com/fluent/fluent-operator/pkg/utils"
+)
+
+// +kubebuilder:object:generate:=true
+
+// An output plugin to submit Prometheus Metrics using the remote write protocol
+type PrometheusRemoteWrite struct {
+	// IP address or hostname of the target HTTP Server, default: 127.0.0.1
+	Host string `json:"host"`
+	// Basic Auth Username
+	HTTPUser *plugins.Secret `json:"httpUser,omitempty"`
+	// Basic Auth Password.
+	// Requires HTTP_user to be se
+	HTTPPasswd *plugins.Secret `json:"httpPassword,omitempty"`
+	// TCP port of the target HTTP Serveri, default:80
+	// +kubebuilder:validation:Minimum:=1
+	// +kubebuilder:validation:Maximum:=65535
+	Port *int32 `json:"port,omitempty"`
+	// Specify an HTTP Proxy. The expected format of this value is http://HOST:PORT.
+	Proxy string `json:"proxy,omitempty"`
+	//Specify an optional HTTP URI for the target web server, e.g: /something ,default: /
+	Uri string `json:"uri,omitempty"`
+	//Add a HTTP header key/value pair. Multiple headers can be set.
+	Headers map[string]string `json:"headers,omitempty"`
+	//Log the response payload within the Fluent Bit log,default: false
+	LogResponsePayload *bool `json:"logResponsePayload,omitempty"`
+	//This allows you to add custom labels to all metrics exposed through the prometheus exporter. You may have multiple of these fields
+	AddLabels []string `json:"addLabels,omitempty"`
+
+	//to do ?
+	Match string `json:"match,omitempty"`
```

Comment:
Why do you want to add this field?
It's a repeat of the one here.
https://github.com/fluent/fluent-operator/blob/master/apis/fluentbit/v1alpha2/clusteroutput_types.go#L38

Reply:
I'll delete it